pkg/config/config.go

```diff
@@ -139,18 +138,17 @@ const (
 	AuthOIDCInitialGroupsClaimName = "auth.oidc.initial_groups_claim_name"
 	AuthLogoutRedirectURL          = "auth.logout_redirect_url"
 
-	BlockstoreTypeKey                    = "blockstore.type"
-	BlockstoreLocalPathKey               = "blockstore.local.path"
-	BlockstoreDefaultNamespacePrefixKey  = "blockstore.default_namespace_prefix"
```

Comment:
OT: Did we delete that?  Isn't this how we recommend a namespace prefix to users, for instance in the playground?

Reply:
Didn't remove the feature - the const key values are usually used for settings defaults.
Didn't find the use of that constant - the default is empty.